repl/plugins/pretty_print.lua

```diff
@@ -110,7 +111,7 @@ local function cross_type_order(a, b)
   local pos_b = type_order[ type(b) ]
 
   if pos_a == pos_b then
-    return a < b
```

Comment:
this isn't working in luajit for me. 

    $ luajit
    > r = require'repl.console'
    > r:loadplugin'pretty_print'
    > r:run()
    > f = {}
    > f[pairs] = 23
    > f[ipairs] = 42
    > return f
    /usr/local/share/lua/5.1/repl/plugins/pretty_print.lua:114: attempt to compare two function values